src/lib/passport/lib/app/index.js

```diff
@@ -4,64 +4,80 @@
 // throwing edge case errors that our API returns, and more.
 //
 
-const express = require('express');
-const csrf = require('csurf');
-const passport = require('passport');
-const app = express();
-const opts = require('../options');
+const express = require("express")
```

Comment:
Ooh, @jacobherrington - if it is not too late (and its totally fine otherwise) can we open a PR that runs Prettier over `src/lib/passport`
```
yarn prettier src/lib/passport/* --write
```
and then rebase into this PR? With these cosmetic changes its making it difficult to see what the actual changes are.

Reply:
(Or actually, lemme do that real quick since its my bad! one sec...)

Reply:
done and merged: https://github.com/artsy/force/pull/9290

Reply:
Cool! I rebased and fixed the merge conflicts.